test.js

```diff
@@ -1,22 +1,16 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var taskkill = require('./');
+import childProcess from 'child_process';
+import test from 'ava';
+import fn from './';
```

Comment:
But `m` does not really makes sense of its a function? Maybe `m` if it's a module and `fn` if it's a function?